### PR TITLE
Readme overhauled

### PR DESCRIPTION
**Pull request by chrisbobo**
@jtrinklein

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,60 @@
...
+
+zshell is recommended. The fish shell has been problematic.
+
+1. `cd app; npm install`
+1. Install Virtualbox (tested on 5.0.6)
+1. Install Docker (this is the hard part)
+    1. `brew install docker` OR download standalone installer
+    1. `sudo chown -R [youruser]:staff ~/.docker`
+    1. `docker-machine create --driver virtualbox dev`
+    1. `eval $(docker-machine env dev)` // Why is this run?
```
Comment (jtrinklein) [inline README.md:13]: `docker-machine env default` is a command whose output is a script which, when run, sets the correct variables in the environment.

`eval $(SOMETHING)` will evaluate the expression `SOMETHING` in the context of the current shell.

combining the two you have a script generated which is then evaluated (executed) within the context of the current shell. End result being that you have the correct variables set in your current shell session.

File: README.md [hunk 2]
```diff
@@ -1,32 +1,60 @@
...
-- Virtualbox - this could probably change
-- helpful script (drop in your bashrc). sets env and echos the ip of the new container
+## Prerequisites
+
+zshell is recommended. The fish shell has been problematic.
+
+1. `cd app; npm install`
+1. Install Virtualbox (tested on 5.0.6)
+1. Install Docker (this is the hard part)
+    1. `brew install docker` OR download standalone installer
```
Comment (jtrinklein) [inline README.md:10]: should just be:
1. Follow install directions on [docker site](https://docs.docker.com/installation/mac/).

File: README.md (hunk 1 above)
Comment (chrisbobo) [inline README.md:13]: Thanks for the explanation. I should've removed the question.

File: README.md [hunk 3]
```diff
@@ -1,32 +1,60 @@
...
+    1. `sudo chown -R [youruser]:staff ~/.docker`
+    1. `docker-machine create --driver virtualbox dev`
+    1. `eval $(docker-machine env dev)` // Why is this run?
+1. `docker-machine env dev`
```
Comment (jtrinklein) [inline README.md:14]: this is redundant and only generates the script net result change is nil

File: README.md [hunk 4]
```diff
@@ -1,32 +1,60 @@
...
+1. `docker-machine ip dev`
```
Comment (jtrinklein) [inline README.md:15]: we should be using `default` for the machine name. it is docker's standard now.

File: README.md (hunk 1 above)
Comment (jtrinklein) [inline README.md:13]: no problem :+1:

File: README.md [hunk 5]
```diff
@@ -1,32 +1,60 @@
...
+1. Add [hosts](/etc/hosts) entry `<ip address from previous command> docker-dev`
```
Comment (jtrinklein) [inline README.md:16]: I'm wondering if there's a utility we can use to add this entry

**Comment by jtrinklein** [comment]
closing as it appears nobody is working on this right now.

**Pull request closed without merging**